src/Models/CheckPush.cs

```diff
@@ -22,6 +22,12 @@ public class AppCheckPushRequest
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "skip_devices")]
         public bool? SkipDevices { get; set; }
 
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "push_provider_name")]
+        public string PushProviderName { get; set; }
+
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "push_provider_type")]
+        public string PushProviderType { get; set; }
```

Comment:
This pr must be merged first: https://github.com/GetStream/stream-chat-net/pull/102/files

```suggestion
        public PushProviderType PushProviderType { get; set; }
```

Reply:
✅